compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -972,6 +972,7 @@ private fun pointerInputEvent(
                 it.pressed,
                 it.pressure,
                 it.type,
+                issuesEnterExit = it.type == PointerType.Mouse,
```

Comment:
Why do we need `issuesEnterExit` field at all? It seems that `type == Mouse` should be enough.

If we really need it, I suggest adding a doc comment with the reason why it's needed and probably rename to something like `issuesHover`

Reply:
It isn't a new field, it was added by Google in commonMain and it means that the pointer can move between nodes and generate Enter/Exit. It exists and used by the internal common code, so we have to set it anyway.

As for checking `type == Mouse` instead, it is always better to check a characteristic of the pointer, not a specific type of it. Imagine, we add a new pointer `VR Controller`, we have to add it everywhere where we added Mouse. Stylus also can support hover without pressing the screen.

I don't like `issuesEnterExit`, `issuesHover`/`always` sounds better, but we already have it, so we can't change it easely. Added a comment in SyntheticEventSender.

The same applicable for our other checks. For example, for primary press checks we should add something like `isPrimaryButtonCausesDown` (naming is hard).